custom-commands/demos/scripts/deployAll.ps1

```diff
@@ -97,16 +97,25 @@ Write-Host -ForegroundColor Yellow "Calling deployTemplate"
 Write-Host -ForegroundColor Yellow "$command"
 Invoke-Expression $command
 
+# some buffer time between each step
+Start-Sleep -s 60
```

Comment:
If you need to add such a long sleep, please print an appropriate message before the sleep so users do not think there is some sort of a hang in the script. 
But in general, this is odd, as I run this script multiple times and did not need this. Isn't 60 seconds way too long?

Reply:
Updated. Only kept sleep period after creating container.